examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala

```diff
@@ -23,6 +33,9 @@ import org.apache.spark.{SparkConf, SparkContext}
 import org.apache.spark.mllib.classification.{NaiveBayes, NaiveBayesModel}
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.regression.LabeledPoint
+
+import java.io.File;
```

Comment:
These imports don't seem to be ordered in the way we would want them to be. You should check out https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark and https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide